src/workerd/jsg/modules.capnp

```diff
@@ -18,6 +18,7 @@ struct Module {
     src @1 :Data; # JS / TS code
     wasm @4 :Data; # Wasm module
     data @5 :Data; # Binary data module
+    json @6 :Text; # Json module
```

Comment:
This is nice, but I wonder if there is a reason we haven't introduced this in the past. We'll have to add this in edgeworker too I suppose and I want to make sure the team is happy with this direction (just calling it out here so hopefully if someone has objections they can call them out)

Reply:
Well I didn't introduce this in the wasm/data PRs because I wasn't including `pyodide-package.json` at all then. It seems much preferable to include them as `json` than to try to do some brittle rewrite on the source.